spec/controllers/items_controller_spec.rb

```diff
@@ -445,6 +449,15 @@
         expect(response.code).to eq('200')
       end
 
+      it 'remove a collection if a new one is not selected' do
```

Comment:
Can you put this in a context block?

e.g. `context 'when a new collection is not selected' do ...`